src/components/Notifications/defaultStyle.ts

```diff
@@ -7,22 +7,41 @@ export default css`
 	.notification-container-bottom-center,
 	.notification-container-bottom-left,
 	.notification-container-bottom-right,
+	.notification-container-center,
```

Comment:
Not sure if this is from the latest version of the notifications library, just make sure to remove node modules and reinstall.

Reply:
Good catch @sradevski fixed it :)